sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseDataView.java

```diff
@@ -139,11 +139,14 @@ protected void initView(String params) {
 		sormasToSormasLocLayout.setMargin(false);
 		sormasToSormasLocLayout.setSpacing(false);
 
-		SormasToSormasListComponent sormasToSormasListComponent = new SormasToSormasListComponent(caze);
-		sormasToSormasListComponent.addStyleNames(CssStyles.SIDE_COMPONENT);
-		sormasToSormasLocLayout.addComponent(sormasToSormasListComponent);
+		boolean sormasToSormasEnabled = FacadeProvider.getSormasToSormasFacade().isFeatureEnabled();
```

Comment:
Is there another ticket (or mechanism that I missed) that constrains the feature to a certain UserRight?